### ISSUE TITLE
Add note on how to mock to the docs

### ISSUE THREAD

**Issue by darrenburns** [darrenburns/ward#264]
We need a page in the docs that tell users that to mock they should use `from unittest import mock`.

We should note that the `mock.patch` decorators don't currently work, and that we need to use the context managers instead.

**Comment by harshlancer** [comment]
If the issue is still relevant I'd like to contribute .@darrenburns

**Comment by darrenburns** [comment]
It's still relevant so feel free to open a PR, thanks!

**Comment by RamonGiovane** [comment]
Yes it's indeed relevant, I wish to know what is the official way of using mocks with Ward.

**Comment by sarkartanmay393** [comment]
I want to contribute to this issue. Assign me this issue.
